### PR TITLE
Rename minioncontroller to nodecontroller

### PR DESCRIPTION
**Pull request by deyuanix**
Catching up.  Start with a mechanical fix.

### CODE REVIEW

File: cmd/integration/integration.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/testapi"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/apiserver"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
-	minionControllerPkg "github.com/GoogleCloudPlatform/kubernetes/pkg/cloudprovider/controller"
+	nodeControllerPkg "github.com/GoogleCloudPlatform/kubernetes/pkg/cloudprovider/controller"
```
Comment (smarterclayton) [inline cmd/integration/integration.go]: would be better as `nodecontroller` (golang convention generally avoids case in pkg names)

**Comment by smarterclayton** [comment]
Just the one comment

File: cmd/integration/integration.go (hunk 1 above)
Comment (thockin) [inline cmd/integration/integration.go]: Given that other places do this, I would not sweat him for this PR.

**Comment by thockin** [comment]
LGTM - merging.  We can fix the multiple bumpyCaps pkg aliases in a followup

**Pull request merged**